junifer/markers/__init__.py

```diff
@@ -23,3 +23,12 @@
     TemporalSNRParcels,
     TemporalSNRSpheres,
 )
+from .complexity import (
```

Comment:
To prevent having `neurokit2` as a requirement, I would not import this here.

The idea is that the user specifies `with: junifer.markers.complexity` in the YAML to use this markers.

The other solution would be to use local imports, but that will fail at runtime if the dependencies are not met.

Reply:
So what about importing it automatically if `neurokit2` is found and not importing it and warning when not found? Something like:
```
if importlib.util.find_spec("neurokit2") is None:
    # don't import and just warn
    warn(...)
else:
    # import complexity markers
```